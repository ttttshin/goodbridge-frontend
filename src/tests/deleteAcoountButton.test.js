import React from "react";
import { render } from "@testing-library/react";
import DeleteAccountButton from "../components/utils/deleteAccountButton";
import { BrowserRouter } from "react-router-dom";

describe("Button component", () => {
  test("renders a button with given label", () => {
    const { asFragment } = render(
      <BrowserRouter>
        <DeleteAccountButton label="Click me" />
      </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
