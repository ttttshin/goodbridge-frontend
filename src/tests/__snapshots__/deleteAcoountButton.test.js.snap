// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders a button with given label 1`] = `
<DocumentFragment>
  <div
    class="shadow-xl bg-offwhite2 rounded-lg p-3 top-10 hover:scale-105 text-center my-3"
  >
    <button>
      Delete Account
    </button>
  </div>
</DocumentFragment>
`;
